src/platform_impl/orbital/mod.rs

```diff
@@ -261,3 +266,17 @@ impl VideoMode {
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct KeyEventExtra {}
+
+impl KeyEventExtModifierSupplement for KeyEvent {
+    #[inline]
+    fn text_with_all_modifiers(&self) -> Option<&str> {
+        //TODO: support modifiers changing text on orbital
+        self.text.as_deref()
+    }
+
+    #[inline]
+    fn key_without_modifiers(&self) -> Key {
+        //TODO: support modifiers changing key on orbital
+        self.logical_key.clone()
+    }
+}
```

Comment:
If it's the same there's no need to implement it.

Reply:
This is for users of that trait (iced) that require it to be implemented. Since Redox doesn't supply different text when control is pressed it mostly does the right thing to provide the original event text and key.

Reply:
How `shift` affects the key? For example `shift+4` should be `$`, however key without modifier is `4`, thus to have some sort of platform parity it should at least emit lowercase latter and handle such cases correctly? 

Or the input is not there yet, so it doesn't really matter?

Reply:
Besides, iced should have a fallback path, since generally you don't have such information, on ios/android, etc. Though, maybe working there is not a priority for them.

Reply:
I will take some time to implement it. Iced doesn't care about Android or IOS at this time.

Reply:
I mean, the whole point of key_without_modifier is that you may want original unshifted key and you design your bindings around those keys, so if redox won't do that, it's literally not different than having a `cfg` and asking for the right key.

I'm not completely against having such impls for redox, because the platform is _special_, however it'll likely yield not really working result.

Key in mind that `ctrl+c` stuff with `text_with_all_modifiers` should likely generate proper C0 codes.

Reply:
I've now implemented these for orbital.